source/docs/03-metadata.xml

```diff
@@ -900,15 +900,16 @@
                      <specDesc key="work"/>
                   </specList>
                </p>
-               <p>All the components of <gi scheme="MEI">work</gi> are optional, but they must occur in the following order:</p>
+               <p>All the components of <gi scheme="MEI">work</gi> are optional. Before MEI 6.0 they had to occur in the following order:</p>
                <list rend="numbered">
                   <item>
                      <gi scheme="MEI">identifier</gi>
                   </item>
                   <item>
                      <gi scheme="MEI">title</gi>
                   </item>
-                  <item>Responsibility-like elements including <gi scheme="MEI">composer</gi>, <gi scheme="MEI">lyricist</gi>, <gi scheme="MEI">contributor</gi> and others</item>
+                  <item>
+                     <gi scheme="MEI">respStmt</gi> (before MEI 6.0: Responsibility-like elements (e.g.,  composer, lyricist and others)</item>
```

Comment:
@riedde There are two opening parentheses "( ... (", but only one is closing ")"